nlpia_bot/etl/scrape_wikipedia.py

```diff
@@ -87,18 +93,34 @@ def load(
         self.df_titles = df
         return self.df_titles
 
-    def find_similar_titles(self, title=None, n=1):
-        """ Takes dot product of a doc vector with all wikipedia title doc vectors to find closest article titles """
-        if isinstance(title, str):
-            vec = nlp(title).vector
-        else:
-            vec = title
-        vec /= pd.np.linalg.norm(vec) or 1.
-        dot_products = vec.dot(self.df_vectors.values.T)
-        if n == 1:
-            return self.df_titles.index.values[dot_products.argmax()]
-        sorted(dot_products, reverse=True)
-
+    #Not currently in use
+    #  def find_similar_titles(self, title=None, n=1):
+    #     """ Takes dot product of a doc vector with all wikipedia title doc vectors to find closest article titles """
+    #     if isinstance(title, str):
+    #         vec = nlp(title).vector
+    #     else:
+    #         vec = title
+    #     vec /= pd.np.linalg.norm(vec) or 1.
+    #     dot_products = vec.dot(self.df_vectors.values.T)
+    #     if n == 1:
+    #         return self.df_titles.index.values[dot_products.argmax()]
+    #     sorted(dot_products, reverse=True)
+
+    def find_titles(self, query='What is a chatbot?', max_titles=30, ngrams=3)

```

Comment:
Missing a colon at the end of this line so build/import on travisci fails.

Reply:
This should be resolved in the commits below.